### PR TITLE
Translation of docs/gatsby-image

### PR DESCRIPTION
**Pull request by AleksanderSkubala**

### CODE REVIEW

File: docs/docs/api-files.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
-title: API Files
+title: Pliki API
 ---
 
-Gatsby uses 4 files in the root of your project to configure your site and control its behavior. All of these files are optional.
+Gatsby używa 4 plików w głownym katalogu projektu do konfiguracji twojej strony oraz kontroli tego jak się zachowuje. Wszytskie z tych plików są opcjonalne.
```
Comment (mbiesiad) [inline docs/docs/api-files.md]: Wszytskie z tych plików są opcjonalne.
change to:
Wszystkie z tych plików są opcjonalne.
// small typo

File: docs/docs/api-files.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-- [gatsby-browser.js](/docs/api-files-gatsby-browser) - Gives you control over Gatsby's behavior in the browser. For example, responding to a user changing routes, or calling a function when the user first opens any page.
-- [gatsby-node.js](/docs/api-files-gatsby-node) - Allows you to respond to events in the Gatsby build cycle. For example, adding pages dynamically, editing GraphQL nodes as they are created, or performing an action after a build is complete.
-- [gatsby-ssr.js](/docs/api-files-gatsby-ssr) - Exposes Gatsby's server-side rendering process so you can control how it builds your HTML pages.
+- [gatsby-config.js](/docs/api-files-gatsby-config) - Pozwala na użycie pluginów, definiuje typowe dane dla strony oraz zawiera inne ustawienia, które pozwalają integrować warstwę danych GraphQL
+- [gatsby-browser.js](/docs/api-files-gatsby-browser) - Pozwala kontrolować to jak Gatsby zachowuje się w przeglądarce. Na przykład: reagowanie na zmiane podstrony przez użytkownika, lub wywoływanie funkcji gdy użytkownik wejdzie na jakąkolwiek stronę.
```
Comment (mbiesiad) [inline docs/docs/api-files.md]: przez użytkownika, lub wywoływanie funkcji gdy użytkownik
change to:
przez użytkownika lub wywoływanie funkcji, gdy użytkownik
// missing comma before 'gdy' and redundant with 'lub'

File: docs/docs/gatsby-image.md [hunk 3]
```diff
@@ -2,17 +2,17 @@
...
+## Wstępne ustawienie Gatsby Image
 
-To start working with Gatsby Image, install the `gatsby-image` package along with necessary plugins `gatsby-transformer-sharp` and `gatsby-plugin-sharp`. Reference the packages in your `gatsby-config.js` file. You can also provide additional options to [`gatsby-plugin-sharp`](/packages/gatsby-plugin-sharp/) in your config file.
+Aby rozpocząć pracę z Gatsby Image, zainstaluj pakiet `gatsby-image` wraz z niezbędnymi pluginami` gatsby-transformer-sharp` i `gatsby-plugin-sharp`. Odwołaj się do pakietów w pliku `gatsby-config.js`. Możesz także podać dodatkowe opcje dla [`gatsby-plugin-sharp`](/packages/gatsby-plugin-sharp/) w pliku konfiguracyjnym.
 
-A common way to source images is to install and use `gatsby-source-filesystem` to connect your local files, but other source plugins can be used as well, such as `gatsby-source-contentful`, `gatsby-source-datocms` and `gatsby-source-sanity`.
+Typowym sposobem na pozyskiwanie obrazów jest instalacja i używanie `gatsby-source-file` do łączenia plików lokalnych, można również używać innych pluginów źródłowych, takich jak `gatsby-source-contentful`, `gatsby-source-datocms` i `gatsby-source-sanity`.
```
Comment (mbiesiad) [inline docs/docs/gatsby-image.md]: `gatsby-source-file`
change to:
`gatsby-source-filesystem`

File: docs/docs/gatsby-image.md [hunk 4]
```diff
@@ -228,11 +228,11 @@ allImageSharp {
...
 ```
 
-Read more about resized image queries in the [gatsby-plugin-sharp](/packages/gatsby-plugin-sharp/#resize) README.
+Przeczytaj więcej o zapytaniach dla obrazów o zmienionym rozmiarze w [gatsby-plugin-sharp](/packages/gatsby-plugin-sharp/#resize) README.
 
-### Shared query parameters
+### Wspólne parametry zapytania
 
-In addition to `gatsby-plugin-sharp` settings in `gatsby-config.js`, there are additional query options that apply to _fluid_, _fixed_, and _resized_ images:
+Oprócz ustawień `gatsby-plugin-sharp` w` gatsby-config.js`, istnieją dodatkowe opcje zapytań, które dotyczą obrazów _fluid_, _fixed_ i _resized_:
```
Comment (mbiesiad) [inline docs/docs/gatsby-image.md]: w` gatsby-config.js`
change to:
w `gatsby-config.js`
// relocated space

**Review by mbiesiad — CHANGES_REQUESTED** [review]
Mainly small typos, nice to have it fixed.

**Comment by AleksanderSkubala** [comment]
Changes done ;)

**Review by mbiesiad — APPROVED** [review]

**Comment by AleksanderSkubala** [comment]
Is everything right with my PR? 😧

**Comment by AdamSiekierski** [comment]
> Is everything right with my PR? 😧

Yea, but the code owners need to merge it. Give'em some time ;)

**Comment by maciekgrzybek** [comment]
Hey guys sorry, I didn't have time to look into it. I'll check tomorrow.

**Review by maciekgrzybek — APPROVED** [review]

**Comment by mbiesiad** [comment]
> Hey guys sorry, I didn't have time to look into it. I'll check tomorrow.

@maciekgrzybek  Sure, it's nice to have you here again! Take your time, have a nice rest! Best greetings

**Pull request merged**